cves/kernel/CVE-2016-9754.yml

```diff
@@ -237,10 +239,8 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 9b94a8fba501f38368aef6ac1b30e7335252a220
+    note: This seems to be an attempt at fixing the buffer overflow. Howvwer it didnt cover every cause, so overflow was still possible.
```

Comment:
Same comment as above. This is an integer overflow.

Reply:
You're right. The integer in question controls the buffer size for the ring buffer, which caused me to misidentify the issue. Fixed the note.